Model/TemplateResolver.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Snowdog
+ *
+ * @author      Dawid Czaja <dawid.czaja@snow.dog>.
+ * @category
+ * @package
+ * @copyright   Copyright Snowdog (http://snow.dog)
+ */
+
+namespace Snowdog\Menu\Model;
+
+use Magento\Framework\View\Element\Template;
+
+class TemplateResolver
+{
+    protected $templateMap;
```

Comment:
It should be  `private` and it should have a default value `$templateMap = [];`

Reply:
updated